test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest==3.0.4
 coala==0.9.0
+coala-bears==0.9.1

```

Comment:
this installs all bears right? it's way too much overhead, can we just make lint PR and add the installation of coala-bears to travis? people can change manually